test/plugins/test/dragresize.js

```diff
@@ -3,6 +3,12 @@
 
     var assert = chai.assert;
 
+    var skipTestIE = function() {
+        if (window.navigator.userAgent.indexOf('Windows') >= 0) {
```

Comment:
This condition doesn't skip the tests on IE, it skips them on Windows. Please use:
`if (CKEDITOR.env.ie) {`
to skip tests on all IE. You may also skip only some version of IE if the functionality doesn't work only on a particular version of IE.